csrankings.csv

```diff
@@ -7381,7 +7387,7 @@ Ian M. Mitchell,University of British Columbia,https://www.cs.ubc.ca/~mitchell,b
 Ian Mackie,University of Sussex,http://www.sussex.ac.uk/informatics/people/peoplelists/person/212026,NOSCHOLARPAGE
 Ian McKillop,University of Waterloo,https://uwaterloo.ca/public-health-and-health-systems/people-profiles/ian-mckillop,NOSCHOLARPAGE
 Ian McQuillan,University of Saskatchewan,https://www.cs.usask.ca/faculty/mcquillan,NOSCHOLARPAGE
-Ian Miguel,University of St Andrews,https://ianm.host.cs.st-andrews.ac.uk,IS6YSBkAAAAJ
+Ian Miers,University of Maryland - College Park,https://www.cs.umd.edu/people/imiers,21kN_WsAAAAJ

```

Comment:
Visting prof, not eligible for inclusion.

Reply:
I had included Dr. Miers as a newly joined Assistant Professor at the University of Maryland, as per their news article from 2019 https://www.cs.umd.edu/article/2019/06/5-new-professors-join-computer-science-faculty (It says that Dr. Miers is joining in the time period of July 1-15, 2020. Kindly advise what should be done with this data entry.

Reply:
That's fine, thanks.